### PR TITLE
Fix test for tensorsolve

### PR DESCRIPTION
**Pull request by unnonouno**
I found `axes` argument in `tensorsolve` is not tested.

### CODE REVIEW

File: tests/cupy_tests/linalg_tests/test_solve.py [hunk 1]
```diff
@@ -43,25 +43,28 @@ def test_invalid_shape(self):
...
+        self.a = numpy.random.randint(
+            0, 10, size=self.a_shape).astype(self.dtype)
+        self.b = numpy.random.randint(
+            0, 10, size=self.a_shape[:2]).astype(self.dtype)
 
-    @condition.retry(10)
-    def test_solve(self):
-        self.check_x((2, 3, 6), (2, 3))
-        self.check_x((3, 4, 4, 3), (3, 4))
+    @testing.for_float_dtypes(no_float16=True)
```
Comment (okuta) [inline tests/cupy_tests/linalg_tests/test_solve.py]: Please remove this line.

**Comment by unnonouno** [comment]
i fixed it

**Comment by okuta** [comment]
Thank you.
Cloud you resolve conflict?

**Comment by unnonouno** [comment]
done

**Comment by okuta** [comment]
LGTM!

**Pull request merged**